benchmark/utils/generator.hpp

```diff
@@ -132,10 +132,7 @@ struct DefaultSystemGenerator {
     {
         auto res = Vec::create(exec);
         res->read(gko::matrix_data<ValueType, itype>(
-            size,
-            std::uniform_real_distribution<gko::remove_complex<ValueType>>(-1.0,
-                                                                           1.0),
-            get_engine()));
+            size, std::uniform_real_distribution<>(-1.0, 1.0), get_engine()));
```

Comment:
This will potentially give us conversion warnings in the future, but I'm generally fine with it

Reply:
I think it is handled in the get_rand_value,